models/User.model.js

```diff
@@ -1 +1,11 @@
 // User model here
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+
+const userSchema = new Schema({
+    username: {type:String, unique:[true, 'Username has to be Unique']},
+    password: {type:String, unique:[true, 'Password has to be Unique']},
```

Comment:
Same as above, most of the students also used the email... which is not required. Glad that you actually read the instructions :)